process-link/src/main/kotlin/com/ritense/processlink/domain/ActivityTypeWithEventName.kt

```diff
@@ -97,7 +97,7 @@ enum class ActivityTypeWithEventName(
 
     SUB_PROCESS_START(getActivityTypeWithEventNameValue(CAMUNDA_SUB_PROCESS, EVENTNAME_START)),
     AD_HOC_SUB_PROCESS_START(getActivityTypeWithEventNameValue(CAMUNDA_SUB_PROCESS_AD_HOC, EVENTNAME_START)),
-    CALL_ACTIVITY_START_START(getActivityTypeWithEventNameValue(CAMUNDA_CALL_ACTIVITY, EVENTNAME_START)),
+    CALL_ACTIVITY_START(getActivityTypeWithEventNameValue(CAMUNDA_CALL_ACTIVITY, EVENTNAME_START)),
```

Comment:
This is a breaking change. For example, In the database of our GZAC test environment, the table`plugin_process_link` contains the `CALL_ACTIVITY_START_START` type.